.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [ 2.7, 3.6, 3.7, 3.8 ]
+        python-version: [ 3.6, 3.7, 3.8 ]
```

Comment:
Let's add 3.9

Reply:
Also remove 3.6

Reply:
3.6 is EOL in 12/2021, so not quite yet.
3.9 support is in this PR: #3848